platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -114,11 +114,6 @@ public SystemPropertiesAdmin(GuestClaimsHandlerExt guestClaimsHandlerExt)
     super(SystemPropertiesAdminMBean.class);
     this.guestClaimsHandlerExt = guestClaimsHandlerExt;
     configureMBean();
-    try {
-      systemDotProperties = new Properties(systemPropertiesFile);
```

Comment:
We want to remove the `private Properties systemDotProperties;` class field, right?

Reply:
Ah good catch, there was a conflict on that line so I must have missed it!